.travis.yml

```diff
@@ -2,6 +2,9 @@ language: python
 python:
   - "2.7"
   - "3.3"
+  - "3.4"
+  - "3.5"
+  - "pypy3"
```

Comment:
Judging by the name `pypy3` is a major version. Should we lock down to minor version as we do with CPython? (e.g. `3.4`)